cromwell.examples.conf

```diff
@@ -452,6 +452,14 @@ backend {
     #    # Limits the number of concurrent jobs
     #    concurrent-job-limit = 5
     #
+    #    # If an 'exit-code-timeout' value is specified:
+    #    # - When a job has not been alive for longer than this timeout (in seconds)
```

Comment:
Having 'seconds' be part of the variable name would help make this self-documenting.

Reply:
Good one, will fix this